### PR TITLE
Add missing file

### PR DESCRIPTION
**Pull request by JacquesOlivierLachaud**
# PR Description

Fix issue #1675 

# Checklist

- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtal/blob/master/ChangeLog.md) added.
- [ ] No warning raised in Debug mode.
- [ ] All continuous integration tests pass (Github Actions & appveyor)

### CODE REVIEW

File: examples/polyscope-examples/SymmetricConvexExpander.h [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+    PointSet   myM;
+
+    /// True iff the set and its local complement are symmetric.
+    bool myPerfectSymmetry;
+    /// Upper bound on the max distance of perfect symmetry.
+    Integer myPerfectSymmetryRadius;
+    
+  };
+
+    // struct PointComparator {
```
Comment (dcoeurjo) [inline examples/polyscope-examples/SymmetricConvexExpander.h:312]: 👍 
cleanup ?

File: examples/polyscope-examples/SymmetricConvexExpander.h [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+#pragma once
+
+/**
+ * @file SymmetricConvexExpander.h
+ * @author Jacques-Olivier Lachaud (\c jacques-olivier.lachaud@univ-savoie.fr )
+ * Laboratory of Mathematics (CNRS, UMR 5807), University of Savoie, France
+ *
+ * @date 2022/09/02
+ *
+ * Header file for module SymmetricConvexExpander.cpp
```
Comment (dcoeurjo) [inline examples/polyscope-examples/SymmetricConvexExpander.h:26]:
```suggestion
```

File: examples/polyscope-examples/SymmetricConvexExpander.h [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+{
+
+  /////////////////////////////////////////////////////////////////////////////
+  // class SymmetricConvexExpander
+  /**
+   * Description of class 'SymmetricConvexExpander' <p>
+   *
+   * \brief Aim: SymmetricConvexExpander computes symmetric fully
+   * convex subsets of a given digital set.
+   *
```
Comment (dcoeurjo) [inline examples/polyscope-examples/SymmetricConvexExpander.h:59]: Why not including this class to the main DGtal/ subtree?

**Review by dcoeurjo — COMMENTED** [review]
Thx, just few comments.

**Comment by JacquesOlivierLachaud** [comment]
It is completely a work in progress, related to  maximal tangent planes. It lays here just to make easier its compilation (it was waiting for inclusion of PR on full convexity).

**Comment by dcoeurjo** [comment]
https://github.com/DGtal-team/DGtal/pull/1674 fixes the python test issues

**Comment by JacquesOlivierLachaud** [comment]
Self merging.

**Pull request merged**